.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: "Build"
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+       fetch-depth: 0
+    - uses: cachix/install-nix-action@v22
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+    - uses: cachix/cachix-action@v12
+      with:
+        name: nix-dev
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+    - name: Build
+      run: nix-build
+    - name: Copy build from Nix store
+      if: github.event_name == 'pull_request'
+      run: |
+        cp -r -L result real-result
```

Comment:
This is to be able to package what's built in the Nix store as a build artifact.

Since symlinks wouldn't work, this dereferences the symlinks and uploads that.